cpp/src/parquet/arrow/reader.cc

```diff
@@ -856,18 +856,32 @@ Status FileReaderImpl::GetRecordBatchReader(const std::vector<int>& row_groups,
     return Status::OK();
   }
 
+  int64_t num_rows = 0;
+  for (int row_group : row_groups) {
+    num_rows += parquet_reader()->metadata()->RowGroup(row_group)->num_rows();
+  }
```

Comment:
Should this be a helper method on the FileMetaData?

Reply:
I don't think `metadata->RowGroupRowCount(row_group)` is an improvement on `metadata->RowGroup(row_group)->num_rows()`